sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/metadata.yaml

```diff
@@ -12,11 +12,3 @@ scheduling:
   secrets:
   - deploy_target: CINDER_TOKEN
     key: bqetl_addons__cinder_bearer_token
-bigquery:
-  time_partitioning:
-    type: day
-    field: 'date'
```

Comment:
I don't think removing the date partitioning is appropriate since the `query.py` script writes to specific date partitions.

I'd suggest either deleting the existing table and letting the artifact deployment process recreate it with the appropriate partition settings, or manually recreating it with the appropriate partition settings.

Reply:
But I do need to get rid of this section too. I can't get the partitions to work right as the date field I want to use is a timestamp and not a date.